[fireplace.__init__]: Initializing card database
[fireplace.__init__]: Merged 2253 cards
[fireplace.entity]: Setting up game Game(players=(Player(name='Player1', hero=None), Player(name='Player2', hero=None)))
[fireplace.entity]: Tossing the coin... Player2 wins!
[fireplace.actions]: Player(name='Player1', hero=<Hero ("Gul'dan")>) triggering <TargetedAction: Summon(<Summon.CARD>=<HeroPower ('Life Tap')>)> targeting [Player(name='Player1', hero=<Hero ("Gul'dan")>)]
[fireplace.actions]: Player1 summons [<HeroPower ('Life Tap')>]
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
[fireplace.actions]: Player(name='Player1', hero=<Hero ("Gul'dan")>) triggering <TargetedAction: Summon(<Summon.CARD>=<Hero ("Gul'dan")>)> targeting [Player(name='Player1', hero=<Hero ("Gul'dan")>)]
[fireplace.actions]: Player1 summons [<Hero ("Gul'dan")>]
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
[fireplace.entity]: Player(name='Player1', hero=<Hero ("Gul'dan")>) shuffles their deck
[fireplace.card]: <Minion ("Spawn of N'Zoth")> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.card]: <Minion ("C'Thun")> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.card]: <Minion ('Mogor the Ogre')> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.card]: <Minion ('Crowd Favorite')> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.actions]: Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>) triggering <TargetedAction: Summon(<Summon.CARD>=<HeroPower ('Dagger Mastery')>)> targeting [Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>)]
[fireplace.actions]: Player2 summons [<HeroPower ('Dagger Mastery')>]
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
[fireplace.actions]: Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>) triggering <TargetedAction: Summon(<Summon.CARD>=<Hero ('Valeera Sanguinar')>)> targeting [Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>)]
[fireplace.actions]: Player2 summons [<Hero ('Valeera Sanguinar')>]
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
[fireplace.entity]: Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>) shuffles their deck
[fireplace.card]: <Minion ('Master Swordsmith')> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.card]: <Minion ('Gnomeregan Infantry')> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.card]: <Minion ('Silverback Patriarch')> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.entity]: Entering mulligan phase
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
[fireplace.entity]: Player(name='Player1', hero=<Hero ("Gul'dan")>) shuffles their deck
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
[fireplace.entity]: Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>) shuffles their deck
[fireplace.entity]: Player1 gets The Coin (GAME_005)
[fireplace.actions]: Player(name='Player1', hero=<Hero ("Gul'dan")>) triggering <TargetedAction: Give(<Give.CARD>='GAME_005')> targeting [Player(name='Player1', hero=<Hero ("Gul'dan")>)]
[fireplace.actions]: Giving [<Spell ('The Coin')>] to Player1
[fireplace.card]: <Spell ('The Coin')> moves from <Zone.SETASIDE: 6> to <Zone.HAND: 3>
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
[fireplace.entity]: Player2 begins turn 1
[fireplace.entity]: Player2 is now at 1 mana crystals
[fireplace.actions]: Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>) triggering <TargetedAction: Draw()> targeting [Player(name='Player2', hero=<Hero ('Valeera Sanguinar')>)]
[fireplace.entity]: Player2 draws <Minion ('Anubisath Sentinel')>
[fireplace.card]: <Minion ('Anubisath Sentinel')> moves from <Zone.DECK: 2> to <Zone.HAND: 3>
[fireplace.entity]: Empty stack, refreshing auras and processing deaths
Traceback (most recent call last):
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 268, in _getattribute
    obj = getattr(obj, subpath)
AttributeError: module 'fireplace.cards' has no attribute 'OG_256'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 907, in save_global
    obj2, parent = _getattribute(module, name)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 271, in _getattribute
    .format(name, obj))
AttributeError: Can't get attribute 'OG_256' on <module 'fireplace.cards' from '/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/fireplace-0.1-py3.5.egg/fireplace/cards/__init__.py'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "multi_test.py", line 65, in <module>
    main()
  File "multi_test.py", line 48, in main
    gamecopy = manager.Game(game.player1, game.player2)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/multiprocess/managers.py", line 628, in temp
    token, exp = self._create(typeid, *args, **kwds)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/multiprocess/managers.py", line 522, in _create
    id, exposed = dispatch(conn, None, 'create', (typeid,)+args, kwds)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/multiprocess/managers.py", line 79, in dispatch
    c.send((id, methodname, args, kwds))
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/multiprocess/connection.py", line 206, in send
    self._send_bytes(ForkingPickler.dumps(obj))
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/multiprocess/reduction.py", line 53, in dumps
    cls(buf, protocol).dump(obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 408, in dump
    self.save(obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 740, in save_tuple
    save(element)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 725, in save_tuple
    save(element)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 520, in save
    self.save_reduce(obj=obj, *rv)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 623, in save_reduce
    save(state)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/dill/dill.py", line 835, in save_module_dict
    StockPickler.save_dict(pickler, obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 810, in save_dict
    self._batch_setitems(obj.items())
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 836, in _batch_setitems
    save(v)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 520, in save
    self.save_reduce(obj=obj, *rv)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 623, in save_reduce
    save(state)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/dill/dill.py", line 835, in save_module_dict
    StockPickler.save_dict(pickler, obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 810, in save_dict
    self._batch_setitems(obj.items())
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 836, in _batch_setitems
    save(v)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 725, in save_tuple
    save(element)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 520, in save
    self.save_reduce(obj=obj, *rv)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 623, in save_reduce
    save(state)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/dill/dill.py", line 835, in save_module_dict
    StockPickler.save_dict(pickler, obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 810, in save_dict
    self._batch_setitems(obj.items())
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 836, in _batch_setitems
    save(v)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 520, in save
    self.save_reduce(obj=obj, *rv)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 617, in save_reduce
    self._batch_appends(listitems)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 794, in _batch_appends
    save(x)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 520, in save
    self.save_reduce(obj=obj, *rv)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 623, in save_reduce
    save(state)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/dill/dill.py", line 835, in save_module_dict
    StockPickler.save_dict(pickler, obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 810, in save_dict
    self._batch_setitems(obj.items())
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 836, in _batch_setitems
    save(v)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 520, in save
    self.save_reduce(obj=obj, *rv)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 623, in save_reduce
    save(state)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/dill/dill.py", line 835, in save_module_dict
    StockPickler.save_dict(pickler, obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 810, in save_dict
    self._batch_setitems(obj.items())
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 836, in _batch_setitems
    save(v)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 475, in save
    f(self, obj) # Call unbound method with explicit self
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/site-packages/dill/dill.py", line 1231, in save_type
    StockPickler.save_global(pickler, obj)
  File "/home/dillon/anaconda2/envs/hearth/lib/python3.5/pickle.py", line 911, in save_global
    (obj, module_name, name))
_pickle.PicklingError: Can't pickle <class 'fireplace.cards.OG_256'>: it's not found as fireplace.cards.OG_256
